lib/new_relic/agent/database_adapter.rb

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+module NewRelic
+  module Agent
+    class DatabaseAdapter
+      VERSIONS = {
```

Comment:
The idea here is that this hash defines the boundaries where a breaking change was made in the adapter lookup implementation in ActiveRecord. For example, since there was no change in implementation between Rails 4.0 and Rails 6.0, they can all use the implementation provided by `'4.0.0'`